03_Modules/Configuration/src/module/module.ts

```diff
@@ -405,14 +419,34 @@ export class ConfigurationModule extends AbstractModule {
     this._logger.debug('Heartbeat response sent: ', messageConfirmation);
   }
 
+  @AsHandler(OCPPVersion.OCPP1_6, OCPP1_6_CallAction.Heartbeat)
```

Comment:
It would be great to move the ocpp 1.6 handler at the end of ocpp 2.0.1 handlers.